tiled/server/router.py

```diff
@@ -617,3 +617,18 @@ async def put_dataframe_full(
             status_code=405, detail="This path cannot accept dataframe data."
         )
     return json_or_msgpack(request, None)
+
+
+@router.put("/node/metadata/{path:path}")
+async def put_metadata(
+    request: Request,
+    body: schemas.PutMetadataRequest,
+    entry=Security(entry, scopes=["write:data", "write:metadata"]),
+):
+    if hasattr(entry, "put"):
+        entry.put_metadata()
```

Comment:
Does this need the metadata and specs as parameters?

Reply:
The right parameters were added in the last commit